loader/windows/icd_windows_dxgk.c

```diff
@@ -121,7 +121,11 @@ bool khrIcdOsVendorsEnumerateDXGK(void)
                 // Try a different value type.  Some vendors write the key as a multi-string type.
                 queryArgs.ValueType = REG_MULTI_SZ;
                 Status = D3DKMTQueryAdapterInfo(&queryAdapterInfo);
-                if (!NT_SUCCESS(Status))
+                if (NT_SUCCESS(Status))
+                {
+                   KHR_ICD_TRACE("Accepting multi-string registry key type\n");
```

Comment:
Inconsistent indentations, use 4 spaces.

Reply:
I will incorporate all of these requested changes and push the changes on Monday (17th).